site.py

```diff
@@ -275,6 +275,11 @@ def container_ready(cid, timeout=DEFAULT_TIMEOUT):
 
     return False
 
+@app.route('/containers/<cid>/logs')
+def logs(cid):
+    stdoutString = docker.logs(container=cid, stdout=True, stderr=False)
```

Comment:
It is not idiomatic in Python (nor most languages) to specify the type in the variable name. `stdout` or just `out` is enough to differentiate with `stderr` or `err`. The more variables in scope the more specific the names, otherwise keep descriptive enough, but short.